upgrade/helpers/tasks.py

```diff
@@ -799,3 +799,33 @@ def pre_upgrade_system_checks(capsules):
     if capsules:
         for capsule in capsules:
             wait_untill_capsule_sync(capsule)
+
+
+def generate_custom_certs():
+    """ Task to generate custom certs for satellite
+    :param: SERVER_HOSTNAME
```

Comment:
There is no param as such, modify this as:

```suggestion
    environment variables:
    
    SERVER_HOSTNAME The satellite server hostname
```